src/coreclr/jit/codegencommon.cpp

```diff
@@ -2213,6 +2213,11 @@ void CodeGen::genEmitMachineCode()
 //
 void CodeGen::genEmitUnwindDebugGCandEH()
 {
+#ifdef TARGET_WASM
+    // TODO-WASM: Fix this phase causing an assertion failure even for methods with no GC locals or EH clauses
```

Comment:
Do you recall what this was? I don't think I see it as a runtime altjit. I guess I can try using crosgen2.